### ISSUE TITLE
Cannot access extensions search result using the keyboard

### ISSUE THREAD

**Issue by kieferrm** [Microsoft/vscode#8661]
- VSCode Version: insiders 7/1
- OS Version: Windows

Steps to Reproduce:

1. Open Extensions
2. Press Tab key
-> Extensions search results get focus, but I can not get to the individual extensions using the arrow keys or the tab key.

**Comment by bpasero** [comment]
I filed a similar bug that the list widget needs to be on par with tree widget in terms of these accessibility features.

**Comment by joaomoreno** [comment]
Yes, closing as duplicate of #8621
